extensions/interactions/MultipleChoiceInput/MultipleChoiceInput.js

```diff
@@ -33,7 +33,7 @@ oppia.directive('oppiaInteractiveMultipleChoiceInput', [
 
         $scope.submitAnswer = function(answer) {
           answer = parseInt(answer, 10);
-          $scope.$parent.$parent.submitAnswer(answer);
+          $scope.$parent.submitAnswer(answer);
```

Comment:
I was just testing and forgot to remove this.

Either $scope.$parent or $scope.$parent.$parent should work, because I remove the isolate scope on the interaction.